src/pages/favorites/favorites.component.spec.ts

```diff
@@ -71,4 +71,37 @@ describe('Favorites Component', () => {
 
     });
   });
+  describe('removeRoute', () => {
+    // Make an arbitrary route object available to each test
+    beforeAll(() => {
+      this.route = {
+        RouteId: 20035,
+        GoogleDescription: 'Campus Shuttle Southbound',
+        ShortName: '35',
+        RouteAbbreviation: '35',
+        Color: '00467E'
```

Comment:
Oh, I found it in [favorite route service](https://github.com/umts/pvta-multiplatform/blob/master/src/providers/favorite-route.service.ts). Isn't this technically a favorite route then? Or do all routes have these properties?